debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ByteCodeDumper.java

```diff
@@ -434,7 +434,7 @@ private static IDATA j9bcutil_dumpBytecodes(PrintStream out, J9ROMClassPointer r
 				out.append(J9UTF8Helper.stringValue(nameAndSig.signature())); /* dump signature */
 				out.append(nl);
 
-				pc = pc.add(4);
+				pc = pc.add(2);
```

Comment:
Would it not be more consistent to instead add `bcIndex = bcIndex.add(2);`?

Reply:
I would rather see the bytecodes that will actually be executed (this is also consistent with cfdump -x).

Reply:
Fair enough.